openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/GlossaryOverviewTab.component.tsx

```diff
@@ -179,16 +179,20 @@ const GlossaryOverviewTab = ({
           </Col>
         </Row>
       </Col>
-      <Col className="p-t-md" span={6}>
-        <GlossaryDetailsRightPanel
-          isGlossary={false}
-          isVersionView={isVersionView}
-          permissions={permissions}
-          selectedData={selectedData}
-          onThreadLinkSelect={onThreadLinkSelect}
-          onUpdate={onUpdate}
-        />
-      </Col>
+      {selectedData.fullyQualifiedName && (
```

Comment:
We should check for whole`selectedData` here as we need that in the right panel.

``` suggestion
      {selectedData && (
```